parsons/etl/etl.py

```diff
@@ -127,13 +127,13 @@ def convert_column(self, *column, **kwargs):
         method. Example usage can be found `here <https://petl.readthedocs.io/en/v0.24/transform.html#petl.convert>`_.
 
         `Args:`
-            \*column: str
+            *column: str
                 A single column or multiple columns passed as a list
-            \**kwargs: str, method or variable
+            **kwargs: str, method or variable
                 The update function, method, or variable to process the update
         `Returns:`
             `Parsons Table` and also updates self
-        """ # noqa: E501,E261
+        """  # noqa: E501,E261
```

Comment:
Same question here as in the other spot where a single-line comment follows a comment block.